foghorn_python/scripts/vcf_comp_funcs.py

```diff
@@ -0,0 +1,138 @@
+import sys
+def GT_comp(GT, lines_holder, sample_index, mis, fields):
```

Comment:
The return values are not used in the calling functions so maybe you don't need to return anything?